app/views/docs/functions-deploy.phtml

```diff
@@ -0,0 +1,216 @@
+    </p>
+</div>
+
+<h3>Manual Deployment</h3>
+<p>You can also upload your functions to be deployed using the Appwrite console. The example below shows a simple Node.JS function, but the same idea applies to any other language.</p>
+
+<div class="ide margin-bottom">
+    <pre class="line-numbers"><code class="prism" data-prism>.
+├── package.json
+└── index.js
+</code></pre>
+</div>
+
+<p>First, navigate inside the folder that contains your dependency file. Package your code files into the <code>.tar.gz</code> format with this tar command:</p>
+
+<ul class="phases clear" data-ui-phases>
+    <li>
+        <h3>Unix</h3>
+
+        <div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
+            <pre class="line-numbers"><code class="prism language-bash" data-prism>tar --exclude code.tar.gz -czf code.tar.gz .</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>CMD</h3>
+
+        <div class="ide margin-bottom" data-lang="bash" data-lang-label="CMD">
+            <pre class="line-numbers"><code class="prism language-bash" data-prism>tar --exclude code.tar.gz -czf code.tar.gz .</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>PowerShell</h3>
+
+        <div class="ide margin-bottom" data-lang="bash" data-lang-label="PowerShell">
+            <pre class="line-numbers"><code class="prism language-bash" data-prism>tar --exclude code.tar.gz -czf code.tar.gz .</code></pre>
+        </div>
+    </li>
+</ul>
+
+<p>Next, navigate to your Appwrite console and upload the function.</p>
+
+<ol class="margin-top margin-bottom-large text-size-normal">
+  <li class="margin-bottom-small">Navigate to the function you want to deploy.</li>
+  <li class="margin-bottom-small">Click <i class="icon-plus"></i> <b>Create deployment</b>.</li>
+  <li class="margin-bottom-small">Select the <b>Manual</b> tab.</li>
+  <li class="margin-bottom-small">Input the entry point of your function under <b>Entrypoint</b>. For the example above, it would be <code>index.js</code>.</li>
+  <li class="margin-bottom-small">Upload <code> code.tar.gz</code>.</li>
+  <li class="margin-bottom-small">Select <b>Activate deployment after build</b> to use your new function.</li>
+  <li class="margin-bottom-small">Click <b>Create</b> to deploy your function.</li>
+</ol>
+
+<h2><a href="#domains" id="domains">Domains</a></h2>
+<p>
+    Each deployed function can have its own domain. 
+    By default, one is generated for each of your functions.
+    You can find the generated domain for your function like this.
+</p>
+
+<ol class="margin-top margin-bottom-large text-size-normal">
+    <li class="margin-bottom-small">Navigate to the Appwrite Console's <b>Functions</b> page.</li>
+    <li class="margin-bottom-small">Navigate to the <b>Domains</b> tab.</li>
+    <li class="margin-bottom-small">In the table, you'll find a link formatted similar to <code>https://64d4d22db370ae41a32e.appwrite.global</code>. This is your preview.</li>
```

Comment:
Lets not call it "preview". THat insicates it cant be used for production, but it can.